pkg/cloud/services/compute/instance.go

```diff
@@ -489,7 +489,17 @@ func (s *Service) DeleteInstance(eventObject runtime.Object, instance *InstanceS
 }
 
 func (s *Service) deletePorts(eventObject runtime.Object, nets []servers.Network) error {
+	trunkSupported, err := s.isTrunkExtSupported()
+	if err != nil {
+		return fmt.Errorf("obtaining network extensions: %v", err)
```

Comment:
I think that is ok to just return the err here, as it is already wrapped with `there was an issue verifying whether trunk support is available`.

Reply:
ok